tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -110,6 +112,7 @@ public interface IHaveInterestingProperties
                 .x(() => exception.Message.Should().Be("The specified expression is not a method call or property getter."));
         }
 
+        [SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object)", Justification = "Not relevant in this case")]

```

Comment:
I guess it would make sense. Unless anyone has a good reason to want to keep it enabled, I'll switch it off.